src/framework/ui/overflowMenu/overflowMenuItem.component.tsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {
+  TouchableOpacity,
+  TouchableOpacityProps,
+  GestureResponderEvent,
+  StyleSheet,
+  ImageProps,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+  Interaction,
+  styled,
+} from '@kitten/theme';
+import {
+  Text as TextComponent,
+  Props as TextProps,
+} from '../text/text.component';
+
+export interface OverflowMenuItemType {
+  icon?: (style: StyleType) => React.ReactElement<ImageProps>;
+  text: React.ReactText;
+  size?: string;
+  isLastItem?: boolean;
+  disabled?: boolean;
+  onPress?: () => void;
```

Comment:
why not to reuse TouchableOpacity `onPress` and `disabled`?

Reply:
Done